src/app/containers/VisuallyHiddenHeadline/index.jsx

```diff
@@ -15,7 +15,7 @@ const VisuallyHiddenHeadlineContainer = ({ blocks }) => {
     return null;
   }
 
-  const id = 'content'; // Used for the skiplink
+  const id = 'headline'; // Used for the skiplink
 
   return (
     <VisuallyHiddenHeadline id={id} tabIndex="-1">
```

Comment:
```suggestion
    <VisuallyHiddenHeadline id="headline" tabIndex="-1">
```

Reply:
What do you think about keeping the headline since we are going to set `aria-labelledby="headline"` on the main wrapper?